package.json

```diff
@@ -524,6 +524,11 @@
         "command": "xml.validation.all.files",
         "title": "Revalidate all opened XML files",
         "category": "XML"
+      },
+      {
+        "command": "xml.command.bind.grammar",
+        "title": "Bind to grammar/schema file",
+        "category": "XML"
```

Comment:
Hide the command when there are no files opened or the current file is not an XML file. (see https://code.visualstudio.com/api/references/when-clause-contexts for a reference on how to do this)